packages/forms/src/Commands/MakeFieldCommand.php

```diff
@@ -67,7 +67,7 @@ public function handle(): int
             $this->copyStubToApp('FieldView', $viewPath);
         }
 
-        $this->components->info("Successfully created {$field}!");
+        $this->components->info("Form Field [$path] created successfully.");
```

Comment:
```suggestion
        $this->components->info("Filament Form Field [$path] created successfully.");
```

Reply:
Loll. The "Filament" word wasnt there originally, so I didnt think of adding them.